### ISSUE TITLE
SQLitePCL.pretty.SQLiteException when ran from D:\ drive.

### ISSUE THREAD

**Issue by FallenArchangel** [MediaBrowser/Emby#2732]
Hello, I've used Emby in the past, but I recently reformated my PC and went to set it up again.  I downloaded the Emby.Windows.zip from the releases page (Version 3.2.20.0) extraced everything to a folder on my Data drive (D:\) and tried to run it.  I got an unhandled exception error and the program crashed every time I tried to launch it. Never completing the first splash loading screen.

I noticed a Disk IO error in the log, so I copied the folder to my Windows drive (C:\) and everything is working normally.
I've ensured I have full rights to the folder on the D:\ drive, but for some reason I can only get emby to launch from C:\.
I have a small SSD on my C:\ drive that I'd rather not use emby from.  If needed I can try symlinking it.

I'm running Windows 10, fully up to date as of this post. 
I7-6700K processor
32Gbs RAM
Nvidia GTX 1070 (8gb vRAM)
C:\ is a 225gb SSD
D:\  is a 2tb HDD

[server-63634268512.txt](https://github.com/MediaBrowser/Emby/files/1110452/server-63634268512.txt)
[unhandled_f51a6436-c02a-4b8a-bf82-34db690e25d4.txt](https://github.com/MediaBrowser/Emby/files/1110451/unhandled_f51a6436-c02a-4b8a-bf82-34db690e25d4.txt)

**Comment by LukePulverenti** [comment]
Sounds like you moved files over that had already been corrupted. Locate the data/users.db file and try renaming that. Then start the server.

**Comment by FallenArchangel** [comment]
I tried removing users.db earlier after reading that it could be a corrupt database problem, aside from an extra entry in the log about it being missing, nothing changed.

I just took the .zip and extracted it to C, worked fine.  Deleted that entire folder, then did the same thing with drive D. Same error. A completely fresh build works on C, but a completely fresh build doesn't work on D.

For now I'll just have all my libraries, metadata, and cache on a different folder on D somewhere. That seems to work just fine for me.  Although it does bug me wondering about what could be causing it.
putting it on a flash drive (F:\) worked just fine.
